infra/gcp/ensure-staging-storage.sh

```diff
@@ -137,6 +137,13 @@ for REPO; do
         color 6 "Enabling the container registry API"
         enable_api "${PROJECT}" containerregistry.googleapis.com
 
+        # Enable container analysis APIs
+        color 6 "Enabling the container analysis API"
+        enable_api "${PROJECT}" containeranalysis.googleapis.com
```

Comment:
I thought enabling the containerscanning API automatically enables the containeranalysis API? See https://cloud.google.com/container-registry/docs/enabling-disabling-container-analysis#enable-scanning

Reply:
oh yeah, I remember discussing this. thank you for catching that error!